packages/dialtone-vue2/components/filter_pill/filter_pill.vue

```diff
@@ -0,0 +1,203 @@
+<template>
+  <dt-popover
+    class="d-filter-pill"
+    data-qa="dt-filter-pill"
+    :open="isPopoverOpen"
+    @update:open="isPopoverOpen = $event"
+  >
+    <template #anchor>
+      <div class="d-filter-pill__wrapper">
+        <dt-button
+          data-qa="dt-filter-pill__button"
+          icon-position="right"
+          importance="outlined"
+          kind="muted"
+          :size="size"
+          :aria-busy="loading"
+          :aria-label="loading ? label : undefined"
+          :class="{ 'd-filter-pill--active': active }"
+          :disabled="disabled"
+          @click="isPopoverOpen = true"
+        >
+          <span v-if="!loading">
+            {{ label }}
+          </span>
+          <dt-skeleton
+            v-else
+            :text-option="{ width: loadingSkeletonWidth }"
+          />
+          <template #icon="{ iconSize }">
+            <dt-icon-chevron-down
+              data-qa="dt-filter-pill__icon"
+              class="d-filter-pill__icon"
+              :size="iconSize"
+            />
+          </template>
+        </dt-button>
+        <dt-button
+          v-if="showReset"
+          data-qa="dt-filter-pill__reset-button"
+          class="d-filter-pill__reset"
+          circle
+          importance="clear"
+          kind="muted"
+          :size="size"
+          :aria-label="resetButtonAriaLabel"
+          @click="$emit('reset', $event)"
+        >
+          <template #icon="{ iconSize }">
+            <dt-icon-close :size="iconSize" />
+          </template>
+        </dt-button>
```

Comment:
![Screenshot 2024-08-28 at 10 20 02 AM](https://github.com/user-attachments/assets/ef7a84e4-5950-401b-918a-e82c9722f72f)

Size of the button feels really big and actually overlaps the text at XL size

Reply:
The button is the same size as the other button, should be ok. What they're doing on product is only changing the "X" icon color but not changing the background color on hover. I can do the same here but I wanted to keep the button implementation.

I thinks this might change with the upcoming @francisrupert updates

Reply:
Preview of [WIP Design](https://dialtone.dialpad.com/deploy-previews/pr-484/components/filter-pill.html), via Draft PR https://github.com/dialpad/dialtone/pull/484.

Reply:
I think I should wait until you have the final HTML implementation so I can make the appropriate changes here.

Maybe closing this PR and open a new one with the final implementation, what do you think? @francisrupert @braddialpad

Reply:
sure, or you can just PR a new PR into this branch.